PHPFIleHandlingAndSessions/createBook.php

```diff
@@ -0,0 +1,32 @@
+<?php
+	session_start();
+	$username=$_SESSION['username'];
+	if(!isset($_SESSION['username']))
+	{
+		@header("location:loginScreen.html");
+	}
+	else
+	{
+		$content=$_POST['content'];
+		$bookName=$_POST['bookName'];
+		try
+		{
+			$db=new PDO('mysql:host=localhost;port=3306;dbname=BooksDetails','root','root');
+			$sql="insert into books(bookName,username) values(:bookName,:username)";
```

Comment:
DB table and table's column names should be underscore separated. Check it everywhere